.github/workflows/ci.yml

```diff
@@ -79,4 +79,4 @@ jobs:
         if: ${{ matrix.python-version == '3.10' }}
         with:
           name: documentation
-          path: docs/_build/html/
+          path: buildstockbatch/docs/_build/html/
```

Comment:
Unrelated to this issue?

Reply:
Yes, unrelated. I missed it on a previous PR.